### PR TITLE
[TASK] Add documentation for child node positioning.

### PR DESCRIPTION
**Pull request by beheist**
Add the missing documentation for the positioning of child nodes in the structure tree.

Resolves: NEOS-1629

### CODE REVIEW

**Comment by aertmann** [comment]
@beheist: Hey is your branch rebased? If not, make sure it's in sync with master repo (sync fork).

File: TYPO3.Neos/Documentation/CreatingASite/NodeTypes/NodeTypeDefinition.rst [hunk 1]
```diff
@@ -69,6 +69,19 @@ The following options are allowed:
             # only allow images in this ContentCollection
             'TYPO3.Neos.NodeTypes:Image': TRUE
             '*': FALSE
+  By using ``position``, it is possible to define the order in which child nodes appear in the structure tree.
```
Comment (aertmann) [inline TYPO3.Neos/Documentation/CreatingASite/NodeTypes/NodeTypeDefinition.rst]: even though rendering works, an empty line should be put here to make the separate blocks easier distinguishable

Comment (beheist) [inline TYPO3.Neos/Documentation/CreatingASite/NodeTypes/NodeTypeDefinition.rst]: Done that.

**Comment by beheist** [comment]
I checked my branch out from 2.0 since I wanted the PR to target 2.0 as well. Should I have branched master instead?

**Comment by aertmann** [comment]
That's correct, but depends on when your fork was created. Since the StyleCI tests are failing it seems like your fork is not up to date with the main repository. To fix it you have to update your branch in your fork with to the main repository. And additionally your custom branch. It's the biggest downside of Github and not so intuitive.

See https://help.github.com/articles/syncing-a-fork/

Basically you checkout 2.0 on your own fork, then say ``git pull upstream 2.0`` and ``git push``. Then checkout your custom branch ``git checkout origin/docs-child-position -b docs-child-position`` and rebase it ``git rebase 2.0``. And then force push ``git push -f``.

Additionally you now have two commits instead of one, so doing a ``git rebase -i HEAD~2`` and squash the second commit.

**Comment by aertmann** [comment]
Hey, just saw this change again and realized that it doesn't mention that ``node:repair`` can be run to update child node positions of existing nodes.. @beheist would you be up for adding that?

**Pull request merged**